pkg/monitortests/network/disruptionserviceloadbalancer/monitortest.go

```diff
@@ -281,7 +282,6 @@ func (w *availability) namespaceDeleted(ctx context.Context) (bool, error) {
 	}
 
 	if err != nil {
-		logrus.Errorf("Error checking for deleted namespace: %s, %s", w.namespaceName, err.Error())
```

Comment:
Do you mean to suppress the return false here?  Looks like this just move the logging up but the comment in the jira had me thinking you wanted to keep polling.

Reply:
I suppose it would be return false, nil to keep polling but still log the error?

Reply:
I think I confused myself that a `return false, err` would not stop the polling. but that's actually what was already happening and how we got to this bug.

but, if we `return false, nil` then I'm not sure we have a way to keep polling. 🤔

Reply:
I had to go look it up earlier myself.  I think by returning `false, nil` it will continue to process.  So you could keep the log entry but return nil for the error.  This doesn't accomplish my other suggestion but that was just a thought.

[PollUntilContextTimeout](https://pkg.go.dev/k8s.io/apimachinery/pkg/util/wait#PollUntilContextTimeout)
```
PollUntilContextCancel tries a condition func until it returns true, an error, or the context is cancelled or hits a deadline
```